### PR TITLE
Migration Tutorial: Removed API versions

### PR DESCRIPTION
**Pull request by megian**
Follow up of tasks from #44.

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `how-to`, `tutorial`, `reference`, `explanation`, `cicd`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: docs/modules/ROOT/pages/tutorials/migration.adoc [hunk 1]
```diff
@@ -6,6 +6,85 @@ This document provides an overview of all the tasks required to move your applic
 
 Before starting the migration of your applications from APPUiO Public to APPUiO Cloud, please check the xref:explanation/differences-to-public.adoc[document explaining all the major differences] between both platforms.
 
+== Removed API versions
+
+Several Kubernetes API versions have been removed since Kubernetes 1.11, on which version APPUiO Public was based on.
+This means the manifests from APPUiO Public can be exported, but needs to be updated, before applicable on APPUiO Cloud.
```
Comment (mhutter) [inline docs/modules/ROOT/pages/tutorials/migration.adoc:12]:
```suggestion
This means that manifests from APPUiO Public can be exported, but need to be updated before being applied on APPUiO Cloud.
```

File: docs/modules/ROOT/pages/tutorials/migration.adoc [hunk 2]
```diff
@@ -6,6 +6,85 @@ This document provides an overview of all the tasks required to move your applic
...
+Often just the API of the object has to be changed.
```
Comment (mhutter) [inline docs/modules/ROOT/pages/tutorials/migration.adoc:13]:
```suggestion
Often just the API version or group of an object has to be changed.
```

File: docs/modules/ROOT/pages/tutorials/migration.adoc [hunk 3]
```diff
@@ -6,6 +6,85 @@ This document provides an overview of all the tasks required to move your applic
...
+For some new API versions, a different structure and other default values apply.
```
Comment (mhutter) [inline docs/modules/ROOT/pages/tutorials/migration.adoc:14]:
```suggestion
For some new API versions however, a different structure or other default values apply.
```

**Review by mhutter — CHANGES_REQUESTED** [review]
I would also suggest using `kubect apply --dry-run=server -f ...` for checking for issues, as it will fail on removed and warn on deprecated APIs

**Review by mhutter — APPROVED** [review]

**Review by tobru — APPROVED** [review]

**Review by corvus-ch — APPROVED** [review]

**Pull request merged**